src/main/java/seedu/address/model/recipe/UniqueRecipeList.java

```diff
@@ -79,6 +79,21 @@ public void remove(Recipe toRemove) {
         }
     }
 
+    /**
+     * Deletes the equivalent recipe from the list, sets deleted flag to true.
+     * The recipe must exist in the list.
+     */
+    public void delete(Recipe toDelete) {
+        requireNonNull(toDelete);
+        System.out.println(1);
+        if (!internalList.contains(toDelete)) {
+            throw new RecipeNotFoundException();
+        }
+        Recipe toAdd = toDelete.delete();
```

Comment:
toAdd is can get confusing here, consider another name (deletedRecipe/updatedRecipe/etc)?